perl/t/init.t

```diff
@@ -0,0 +1,13 @@
+use strict;
+use warnings;
+use Test2::V0;
+
+use Nix::Store;
+
+my $s = new Nix::Store("dummy://");
```

Comment:
fwiw perlcritic complains about that and suggests to use `Nix::Store->new()` instead (noticed that while modifying Hydra for that).

Reply:
Fine with me! I don't know Perl enough to disagree :)